apis/database/v1beta1/sql_types.go

```diff
@@ -18,6 +18,7 @@ package v1beta1
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Nit: We import this as `metav1` by convention - see the above line.

Reply:
will fix